wdio-chrome.conf.js

```diff
@@ -271,9 +289,9 @@ exports.config = {
             const reportError = new Error('Could not generate Allure report');
             const generation = allure(['generate', '../../../../test/results/allure', ' --clean']);
 
-            for (let i = 0; i < chromeSettings.reporters.length; i++) {
-                for (let j = 0; j < chromeSettings.reporters[i].length; j++) {
-                    if (chromeSettings.reporters[i].includes('allure')) {
+            for (let i = 0; i < configurationSettings.reporters.length; i++) {
+                for (let j = 0; j < configurationSettings.reporters[i].length; j++) {
+                    if (configurationSettings.reporters[i].includes('allure')) {
                         return new Promise((resolve, reject) => {
                             const generationTimeout = setTimeout(
                                 () => reject(reportError),
```

Comment:
This should be refactored (I know it was there before the PR). What's the objective of this code snippet exactly?
This can be easily achieved with probably a one or two-liner.

Reply:
This checks whether or not `allure` is a valid reporter for an f-* and launches it after the test run if present. @LTurns Are you able to refactor this in a separate PR?

Reply:
I'm happy to help with this, of course! Ping me if needed.